configuration-sample/ods-core.env.sample

```diff
@@ -161,3 +161,54 @@ STORAGE_CLASS_DATA=""
 
 # Storage class for backup, for AWS this should be "gp2-encrypted"
 STORAGE_CLASS_BACKUP=""
+
+
+####################
+# Provisioning App #
+####################
+
+PROV_APP_ENVIRONMENT=cd
+
+# which Docker image of the prov app to run
+PROV_APP_IMAGE=opendevstackorg/ods-provisioning-app:latest
+
+# which image-pull-policy to use 
+# see https://docs.openshift.com/container-platform/3.11/dev_guide/managing_images.html#image-pull-policy
+PROV_APP_IMAGE_PULL_POLICY=Always
+
+# confluence url - including protocol
+PROV_APP_CONFLUENCE_URI=http://192.168.56.31:8090
```

Comment:
I would change this, and the URI's below, to just `CONFLUENCE_URL`, `BITBUCKET_URL`. That way users do not have to specify it twice.